rules/macos/credential_access_credentials_keychains.toml

```diff
@@ -36,7 +36,18 @@ process where event.type in ("start", "process_started") and
       "System.keychain",
       "login.keychain-db",
       "login.keychain"
-    )
+    ) and
+    not process.args : ("find-certificate",
+                      "add-trusted-cert",
+                      "set-keychain-settings",
+                      "delete-certificate",
+                      "/Users/*/Library/Keychains/openvpn.keychain-db",
+                      "show-keychain-info",
+                      "lock-keychain",
+                      "set-key-partition-list",
+                      "import",
+                      "find-identity") and
+    not process.parent.executable : "/Applications/OpenVPN Connect/OpenVPN Connect.app/Contents/MacOS/OpenVPN Connect"
```

Comment:
would you consider eliminating the root user as well? bring the noise down a bit more.

Reply:
access to those folders requires root, if filtered-out may risk to miss stuff.

Reply:
makes sense, i wasn't sure if that was the case. i def saw non root users trigger this as well. but i think the numbers are low enough.